software/rfcs/rocketFind.md

```diff
@@ -0,0 +1,62 @@
+
+This is a faily open ended proposal, since I don't know what the best solution is, nor do I know if I've covered every possible solution to the problem. What I am going to do is discuss several different ideas that we could use to locate the rocket faster.
+
+## General approach
+
+All of the below proposals have the same underlying approach - have something that broadcasts a signal, and use the signal strength to allow people to locate the device. A key advantage of this approach is that it allows us to find the rocket regardless of the network conditions at the landing site, since we're only sending a signal and not sending data over a network. The tradeoff to not connecting to a network is that we'll have to be within a certain distance of the rocket to locate it, which would depend on the type of radio broadcast device we use.
+
+Note: None of these should be integrated with the main avionics, so we can iterate on this, without changing the firmware of the main avionics.
+
+### Approach 1 - Bluetooth
+
+Here our transmitter would be a bluetooth device, these are fairly easy to find off the shelf, and also mean that we won't have to make any special hardware to find the device, since all *modern* mobile phones have bluetooth capabilities, and should be able to give us reports on the signal strength of the device. Normal bluetooth is said to (read: I Googled it) have a range of upto 100m outdoors, and we're probably not gonna land 100m away, what this means is that when searching, we'd need to split up with our phones, and search until someone gets a ping, then we'd converge on the location of whoever got the ping, and *should* be able to triangulate the position of the rocket using them.
+
+I suspect that the actual range might be much higher than 100m since we're launching in the middle of nowhere and shouldn't have too much interference. We can probably get a good estimate of the range by testing in Richmond Park.
+
+We'd also need to ensure that the battery of any bluetooth beacon lasts several hours, just in case we can't find it on the day and need to look for it the next day.
+
+We might also have an issue when locating the device, since the connection is fairly spotty, so we might not be able to actually tell each other that we've got a ping. To rememdy this I suggest we have flares (no not really).
+
+### Approach 2 - Bluetooth Low Energy
+
+This is pretty much Approach 1, but with a longer battery life, and smaller transmission range of ~50m. Because of the shorter transmission range, I suggest we don't exclusively use BLE, but instead use it if we don't find the rocket within say 3 hours. If we end up doing that, we may not be able to use an off the shelf beacon, but instead have to make one ourselves.
+
+
+### Approach 3 - Wifi
+
+This is basically the ones above, but with yet another transmission range/power consumption trade off. Here the rocket would be a wifi hotspot that we'd need to find.
+
+
+### Approach 4 - LoRa
```

Comment:
A combo of this plus a cell tower method is our way forward for larger rockets imo. We can either continute using the curve, buy a better tracker with more bands supported (curve only supports GSM) or use a more capable SIM board with LTE support.

Perhaps a project could be dismantling an android phone to see if we could run it headless?

Reply:
What's "this"? Lora or a hotspot?

As long as the board isn't the sim800L I'm happy. 
Dismantling an android also sounds fun